public/app/plugins/datasource/prometheus/configuration/ExemplarSetting.tsx

```diff
@@ -74,12 +74,37 @@ export default function ExemplarSetting({ value, onChange, onDelete }: Props) {
                 datasourceUid: undefined,
                 name: value.name,
                 url: event.currentTarget.value,
+                urlDisplayLabel: value.urlDisplayLabel,
               })
             }
           />
         </InlineField>
       )}
 
+      {isInternalLink ? (
```

Comment:
I think we should give the option to override the label whether or not the link is an internal one. This is how Loki derived fields works, so let's remove the conditional here. 

Also, that matches the logic in prometheus/result_transformer.